planetiler-core/src/main/java/com/onthegomap/planetiler/Planetiler.java

```diff
@@ -752,6 +764,50 @@ private void ensureInputFilesExist() {
     }
   }
 
+  // Helper for globMatchPath which will not recurse into zip files.
+  private List<Path> globMatchPath(Path basePath, String pattern) {
```

Comment:
nit: instead of inline comment, maybe rename this to something to indicate it doesn't recurse into nested zips?